sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/PageBlobAPITest.groovy

```diff
@@ -201,14 +202,14 @@ class PageBlobAPITest extends APISpec {
         bc.uploadPages(new PageRange().start(0).end(PageBlobClient.PAGE_BYTES * 2 - 1), data)
 
         then:
-        def e = thrown(Exception)
+        def e = thrown(exceptionType)
         exceptionType.isInstance(e)
```

Comment:
Don't need this line anymore, just the `thrown(exceptionType)`

Reply:
Good catch.